src/app/screens/Home/DefaultView/index.tsx

```diff
@@ -164,10 +188,88 @@ const DefaultView: FC<Props> = (props) => {
 
         {!isLoading && (
           <div>
+            <div className="flex flex-col gap-2 md:gap-3">
+              {transactions.length == 0 && (
+                <IconLinkCard
+                  title={t("default_view.actions.get_started.title")}
+                  description={t(
+                    "default_view.actions.get_started.description"
+                  )}
+                  icon={
+                    <PopiconsBulbLine className="w-8 h-8 text-gray-400 dark:text-neutral-500" />
+                  }
+                  onClick={() => {
+                    utils.openUrl(
+                      "https://guides.getalby.com/user-guide/v/alby-account-and-browser-extension/"
+                    );
+                  }}
+                />
+              )}
+
+              {!(
+                currentAccount?.hasMnemonic &&
+                currentAccount?.isMnemonicBackupDone
+              ) && (
+                <IconLinkCard
+                  title={t("default_view.actions.backup_masterkey.title")}
+                  description={t(
+                    "default_view.actions.backup_masterkey.description"
+                  )}
+                  icon={
+                    <PopiconsKeyLine className="w-8 h-8 text-gray-400 dark:text-neutral-500" />
+                  }
+                  onClick={async () => {
+                    if (currentAccount?.hasMnemonic) {
+                      openOptions(
+                        `accounts/${currentAccount?.id}/secret-key/backup`
+                      );
+                    } else {
+                      openOptions(
+                        `accounts/${currentAccount?.id}/secret-key/new`
+                      );
+                    }
+                  }}
+                />
+              )}
+
+              {transactions.length == 0 && (
+                <IconLinkCard
+                  title={t("default_view.actions.receive_bitcoin.title")}
+                  description={t(
+                    "default_view.actions.receive_bitcoin.description"
+                  )}
+                  icon={
+                    <PopiconsArrowDownLine className="w-8 h-8 text-gray-400 dark:text-neutral-500" />
+                  }
+                  onClick={() => {
+                    navigate("/receive");
+                  }}
+                />
+              )}
+
+              {!(
+                currentAccount?.hasMnemonic &&
+                currentAccount?.isMnemonicBackupDone
```

Comment:
This would mean all existing users will see this reminder. Can we change this to being true by default? Existing users should not see this hint.

Reply:
resolved